iml-gui/crate/src/components/paging.rs

```diff
@@ -133,7 +133,11 @@ pub fn dir_toggle_view<T>(dir: Dir, more_attrs: Attrs) -> Node<T> {
     font_awesome(more_attrs, x)
 }
 
-pub fn limit_selection_view(p: &Model) -> Node<Msg> {
+pub(crate) fn limit_selection_view(p: &Model) -> Node<Msg> {
+    if !p.has_pages() {
+        return empty![];
```

Comment:
BTW, I was thinking about not showing paging controls when the number of items is less than the minimum.

Reply:
Seems sensible.